code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -407,7 +407,7 @@
 		if(getorgan(/obj/item/organ/brain))
 			if(!key)
 				msg += "[span_deadsay("[t_He] [t_is] totally catatonic. The stresses of life in deep-space must have been too much for [t_him]. Any recovery is unlikely.")]\n"
-			else if(!client)
+			else if(!client && !fake_client)
 				msg += "[t_He] [t_has] a blank, absent-minded stare and appears completely unresponsive to anything. [t_He] may snap out of it soon.\n"
```

Comment:
Isn't the body supposed to be dead anyways? Reviving it should be a tell no?

Reply:
Body does not die if damage taken is stamina damage